### DISCUSSION TITLE
How to use specific RC versions (before RC-7)

### DISCUSSION THREAD

**Discussion by pi0** [nuxt/nuxt#17046] [General]
For nuxt versions before RC-7, if for any reason you need to stick with a specific RC version of Nuxt, you can use resolutions: (Issue context: https://github.com/nuxt/framework/issues/5621)

package.json: (yarn)

```json
  "resolutions": {
    "nuxt": "3.0.0-rc.4",
    "nuxi": "3.0.0-rc.4",
    "@nuxt/vite-builder": "3.0.0-rc.4",
    "@nuxt/kit": "3.0.0-rc.4",
    "@nuxt/schema": "3.0.0-rc.4"
  }
```

package.json: (npm)

```json
  "overrides": {
    "nuxt": "3.0.0-rc.4",
    "nuxi": "3.0.0-rc.4",
    "@nuxt/vite-builder": "3.0.0-rc.4",
    "@nuxt/kit": "3.0.0-rc.4",
    "@nuxt/schema": "3.0.0-rc.4"
  }
```

package.json (pnpm)

```json
  "pnpm": {
    "overrides": {
      "nuxt": "3.0.0-rc.4",
      "nuxi": "3.0.0-rc.4",
      "@nuxt/vite-builder": "3.0.0-rc.4",
      "@nuxt/kit": "3.0.0-rc.4",
      "@nuxt/schema": "3.0.0-rc.4"
    }
  }
```

**Comment by alpeshvmodi** [comment]
Hey @pi0,

This worked for me.

But if you remove '^' from the package.json in your code base for the above packages and apply required dependency to respective nuxt release. The user would not be needed to override packages.

Thanks,

**Reply by danielroe** [reply]
Releases from rc7 onward do not include carets. 👍

**Reply by alpeshvmodi** [reply]
Wow. Thats Great News. Thanks @danielroe
